rest-client/client-common/src/main/java/io/apicurio/registry/rest/client/exception/GroupNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package io.apicurio.registry.rest.client.exception;
+
+import io.apicurio.registry.rest.v2.beans.Error;
+
+public class GroupNotFoundException extends NotFoundException {
+
+    private static final long serialVersionUID = 1L;
+
+    public GroupNotFoundException(Error error) {
```

Comment:
Same here, do we want to be specific at this level or do we want to just return a 404 with no info about the concrete resource that wasn't found? Perhaps just return a 404 containing the full resource URL that cannot be found? Thoughts?